site-book/bin/generate-md.sh

```diff
@@ -190,7 +190,13 @@ cd site-book/src/site/markdown
 # Rewrite hrefs for resource references, using table provided by Maintainers
 for (( i=0; i<${#HREF_REWRITE_LIST[@]} ; i+=2 )) ; do
     echo rewriting href in "${HREF_REWRITE_LIST[$i]}" : "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}"
-    sed -i '' -e "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}" "${HREF_REWRITE_LIST[$i]}"
+    if [[ "$OSTYPE" =~ "darwin" ]] ; then
+        # Macs need an argument after -i option 
+        sed -i '' -e "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}" "${HREF_REWRITE_LIST[$i]}"
+    else
```

Comment:
I would suggest handling this slightly more explicitly.  My two thoughts are:
1.  Approach it similar to [here](https://github.com/anandsubbu/incubator-metron/blob/dc3fa9739bad53da7931d992cdfd499abba5387d/metron-deployment/scripts/platform-info.sh#L68), using a `case "${OSTYPE}"` to watch `darwin*` and `linux*`, and then a `*)` that alerts the user.
1. By adding an elif for either `"${OSTYPE}" == linux*` or `"${OSTYPE}" =~ "linux"`, and then an `else` that alerts the user.